templates/clockwork.toml.j2

```diff
@@ -34,4 +34,20 @@ allocations="*"
 allocations={{ cluster.allocations }}
 {% endif %}
 timezone="{{ cluster.timezone }}"
+organization="{{ cluster.organization }}"
+nbr_cpus={{ cluster.nbr_cpus }}
+nbr_gpus={{ cluster.nbr_gpus }}
+official_documentation="{{ cluster.official_documentation }}"
+mila_documentation="{{ cluster.mila_documentation }}"
+display_order={{ cluster.display_order }}
+remote_user="{{ cluster.remote_user }}"
+remote_hostname="{{ cluster.remote_hostname }}"
+{% if cluster.sacct_enabled %}
+sacct_enabled={{ cluster.sacct_enabled }}
+{% else %}
+sacct_enabled=false
+{% endif %}
```

Comment:
This doesn't need to be in an if/else construct.

Reply:
Doesn't it?

If the this field is missing, won't the template will just leave it empty so it'll be
```
sacct_enabled=
```
which is going to cause issues with the parsing of the configuration (I'm guessing).

I don't know for sure. I think I modelled that after the `update_field` case that you did, but that situation is a bit different.

Reply:
If this field is missing the render will crash.

`update_field` is different because it is either a string or the boolean value `false` to disable the functionality.

Reply:
When you say that the "render will crash", do you mean that Ansible will catch that error, or that it will propagate elsewhere and cause problems in Clockwork?

Reply:
Ansible will fail in that case.